intelmq/tests/bots/parsers/shadowserver/test_sandbox_conn.py

```diff
@@ -0,0 +1,96 @@
+# SPDX-FileCopyrightText: 2019 Guillermo Rodriguez
```

Comment:
2019? This looks like a new file.

Reply:
This also applies to lots of other files

Reply:
The copyright has been updated on files added.